### PR TITLE
Change Use Case sidebar label

### PR DESCRIPTION
**Pull request by kentdanas**

### CODE REVIEW

**Review by TJaniF — APPROVED** [review]
LGTM! :)

File: sidebarsLearn.js [hunk 1]
```diff
@@ -246,28 +249,19 @@ module.exports = {
     },
     {
       type: 'category',
-      label: 'Examples',
+      label: 'Use Cases',
```
Comment (jwitz) [inline sidebarsLearn.js:252]:
```suggestion
      label: 'Use cases',
```

File: sidebarsLearn.js [hunk 2]
```diff
@@ -246,28 +249,19 @@ module.exports = {
...
       link: {
         type: 'generated-index',
-        title: 'Examples',
-        description: 'See examples of use cases and more with Apache Airflow.'
+        title: 'Use Cases',
```
Comment (jwitz) [inline sidebarsLearn.js:255]:
```suggestion
        title: 'Use cases',
```

**Review by jwitz — APPROVED** [review]
LGTM, makes a lot of sense now that we've moved connection docs out of here. Just one minor comment re: capitalization

**Pull request merged**